cmd/kaspawallet/daemon/server/create_unsigned_transaction.go

```diff
@@ -68,7 +80,9 @@ func (s *server) selectUTXOs(spendAmount uint64, feePerInput uint64) (
 	}
 
 	for _, utxo := range s.utxosSortedByAmount {
-		if !isUTXOSpendable(utxo, dagInfo.VirtualDAAScore, s.params.BlockCoinbaseMaturity) {
+		addr, err := s.walletAddressString(utxo.address)
+		if err != nil || (from != nil && addr != from.String()) ||
```

Comment:
Why do you swallow the error?

Reply:
I thought to ignore invalid UTXOs (which cannot be formatted to address). Maybe a better approach would be to convert `from` to `walletAddress` type and explicitly throw an error if failing. I'll look into it